Source/Core/Common/StringUtil.h

```diff
@@ -8,6 +8,7 @@
 #include <cstddef>
 #include <cstdlib>
 #include <iomanip>
+#include <limits> /* std::numeric_limits */
```

Comment:
I think these comments are unnecessary.
We don't do this anywhere else and no one is going to keep them updated as header requirements change.

Reply:
Agreed. This pull request LGTM if the comments are removed.

Reply:
Sounds good. Re-pushed patches without a comment.